crime-commons-schemas/src/main/resources/schemas/atis/crime_application_result.json

```diff
@@ -0,0 +1,67 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "crime_application_result.json",
+  "type": "object",
+  "properties": {
+    "usn": {
+      "type": "integer",
+      "nullable": true,
+      "description": "USN number"
+    },
+    "maat_ref": {
+      "type": "integer",
+      "nullable": true,
+      "description": "MAAT / Rep Id"
+    },
+    "case_id": {
+      "type": "string",
+      "nullable": true,
+      "description": "Case ID"
+    },
+    "case_type": {
+      "type": "string",
+      "nullable": true,
+      "enum": ["INDICTABLE", "SUMMARY ONLY", "CC ALREADY", "APPEAL CC", "COMMITAL", "EITHER WAY"]
```

Comment:
Is this the spelling of "commital" we want? It would normally be spelt "committal".

Reply:
No, It is a Code we are using in MAAT